src/reducers/MapReducer.js

```diff
@@ -56,16 +58,27 @@ const INITIAL_STATE = {
     viewStyle: map.SearchOptions.MAP,
     sites: [],
     displaySites: [],
+    filterResultsScrutinyLoose: {facilities: true, features: true},
     filterCriteriaKeys: {accessibility: [], facilities: [], price: [], features: []}
 };
 
+const updateFilterResultsScrutiny = ({filterResultsScrutinyLoose}, filterToggleKey) => {
+    let filterResultsScrutinyClone = _.cloneDeep(filterResultsScrutinyLoose);
+
+    filterResultsScrutinyClone[filterToggleKey] = !filterResultsScrutinyClone[filterToggleKey];
+
+    return filterResultsScrutinyClone;
+};
```

Comment:
Since the `Every` and `Any` toggles are just booleans, when we toggle them, the new value should just be the opposite of what it previously was.